operator/src/monitoring/otel-config.yaml

```diff
@@ -0,0 +1,82 @@
+---
+receivers:
+  # Push based metrics
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+  # Pull based metrics
+  prometheus/scrape_configs:
+    config:
+      scrape_configs:
+        - job_name: 'kubernetes-service-endpoints'
+          scrape_interval: 10s
+          scrape_timeout: 1s
+
+          kubernetes_sd_configs:
+          - role: pod
+            namespaces:
+              own_namespace: true
+          # Only container ports named `metrics` will be considered valid targets.
+          #
+          # Setup relabel rules to give meaning to the following k8s annotations:
+          #   prometheus/path - URL path of the metrics endpoint
+          #
+          # Example:
+          #   annotations:
+          #      prometheus/path: "/api/v0/metrics"
+          relabel_configs:
+          - source_labels: [__meta_kubernetes_pod_container_port_name]
+            action: keep
+            regex: "metrics"
+          - source_labels: [__meta_kubernetes_pod_annotation_prometheus_path]
+            action: replace
+            target_label: __metrics_path__
+            regex: (.+)
+          - source_labels: [__meta_kubernetes_namespace]
+            action: replace
+            target_label: kubernetes_namespace
+          - source_labels: [__meta_kubernetes_pod_name]
+            action: replace
+            target_label: kubernetes_pod
+          - source_labels: [__meta_kubernetes_pod_container_name]
+            action: replace
+            target_label: kubernetes_container
+
+processors:
```

Comment:
Can we filter out duplicates while processing?

Reply:
We can if necessary, see this issue about the situation.

https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/14900